pickee_api/models.py

```diff
@@ -52,7 +52,7 @@ class Meta:
         unique_together = ('user', 'actor')
 
     def __str__(self):
-        return self.actor.name
+        return self.user.email + ": " + self.actor.name
```

Comment:
Is it used somewhere in the system or just for demo purposes?

Reply:
It's used in the admin panel